LHM/Problems/beakjoon/1110.py

```diff
@@ -0,0 +1,21 @@
+import math
+inputVal = int(input())
+in1 = math.floor(inputVal/10)   
+in2 = inputVal%10               
+i = 0
+print(in1,in2)
+
+input = []
+input.insert(0,in1)
+input.insert(1,in2)
```

Comment:
What if in1 was list? Then the sentence 10 can't be work properly.
```suggestion
input.assert(in1)
input.assert(in2)
```